session.js

```diff
@@ -0,0 +1,52 @@
+import { isValidApplePayGateway } from './validateApplePayGateway';
+import responseJson from './responseJson';
+import request from 'request';
+
+export const create = async (event, context, callback) => {
+
+    const serverlessRequest = JSON.parse(event.body) || {url: '', body: {}};
+    const endpointURL = serverlessRequest.url || '';
+
+    if (!isValidApplePayGateway(endpointURL)) {
+        callBack(null, responseJson(400, {message: 'Hello Sad World!'}))
+        return
+    }
+    return createSession(endpointURL, serverlessRequest.body);
+};
+
+const createSession = (endpointURL, body) => new Promise((resolve, reject) => {
+
+  const merchIdentityCert = process.env.MERCHANT_IDENTITY_CERT;
+  const merchIdentityKey = process.env.MERCHANT_IDENTITY_KEY;
+
+  const options = {
+    method: 'POST',
+    uri: endpointURL,
+    cert: merchIdentityCert,
+    key: merchIdentityKey,
+    body: body || {},
+    json: true,
+  };
+
+  request(options, function (error, response, body) {
```

Comment:
why not use arrow notation like in the remaining code?

Reply:
fixed thanks